src/radical/pilot/configs/resource_xsede.json

```diff
@@ -357,12 +357,9 @@
         "task_launch_method"          : "SPARK",
         "mpi_launch_method"           : "MPIRUN_RSH",
         "pre_bootstrap_1"             : [ 
-                                        "module purge",
-                                        "module load TACC",
-                                        "module load python/2.7.12",
+                                        "module load python/2.7.9",
                                         "export TACC_DELETE_FILES=TRUE",
```

Comment:
This looks wrong - 2.7.9 is gone, and would not load.  2.7.12 is what we use on stampede, AFAIK.  Is this from a botched merge, maybe?